Firestore/src/Connection/Grpc.php

```diff
@@ -242,6 +254,12 @@ public function runQuery(array $args)
             new StructuredQuery,
             $this->pluck('structuredQuery', $args)
         );
+        if (isset($args['readTime'])) {
+            $args['readTime'] = $this->serializer->decodeMessage(
+                new ProtobufTimestamp(),
+                $args['readTime']
+            );
+        }
```

Comment:
Consider encapsulating this to a private function for reuse

Reply:
Was little bit skeptical about this earlier as very few APIs use `readTime`, but now that I see it, it feels more clean after doing it in commit [41b2311](https://github.com/googleapis/google-cloud-php/pull/5865/commits/41b2311e14606643954616150399eaf1a13d0ba1).